src/v2i-hub/CARMAStreetsPlugin/src/JsonToJ3224SDSMConverter.cpp

```diff
@@ -94,7 +94,11 @@ namespace CARMAStreetsPlugin
         auto ref_pos_ptr = CARMAStreetsPlugin::create_store_shared<Position3D_t>(shared_ptrs);
         ref_pos_ptr->lat = sdsm_json["ref_pos"]["lat"].asInt64();
         ref_pos_ptr->Long = sdsm_json["ref_pos"]["long"].asInt64();
+        #if SAEJ2735_SPEC < 2020
```

Comment:
Do we need to support both versions or upgrade to one version?

Reply:
I'm not really sure. The previous version had this logic to support both, but you could only use one at a time. Do you think we should just make separate tags for different versions or something else?

Reply:
I agree. I think we should at least phase out anything pre-2020. Nobody will be using anything older anyway, so it doesn't make sense to maintain support.

Reply:
I am good with how it is now. Maybe later we can come back with a solution